commands/lxc2lxd.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package commands
+
+import (
+	"bytes"
+	"os"
+	"strings"
+
+	"github.com/juju/description"
+	"github.com/juju/errors"
+)
+
+//go:generate go run ../juju2/generate/filetoconst/filetoconst.go LXCMigrationScript ../../../lxc/lxd/scripts/lxc-to-lxd lxc2lxd_script.go 2017 commands
```

Comment:
> Oh weird, the script was already in the codebase?

The filetoconst code generation helper is there already, which you can reuse. I just ran it and checked in the generated code (lxc2lxd_script.go). As Tim says, it's pointing at the script from the lxc/lxd repo.